pandas/tests/io/excel/test_readers.py

```diff
@@ -67,6 +60,24 @@ def engine(request):
     return request.param
 
 
+@pytest.mark.parametrize("kwargs", [dict(), dict(engine=None)])
+def test_default_engine(datapath, monkeypatch, kwargs):
+    """
```

Comment:
We don't use docstrings for tests so OK to remove this as well. You can add the issue number as a comment instead